hw/mcu/nordic/nrf5340/syscfg.yml

```diff
@@ -126,6 +126,13 @@ syscfg.defs:
             Application code to runs in secure mode.
             This settings must have same value for bootloader and application build.
         value: 1
+    MCU_NET_GPIO:
```

Comment:
hmm there is already MCU_GPIO_NET

Reply:
Thanks, old MCU_GPIO_NET reused. Old code from startup that used it is now removed.